### PR TITLE
chore: Improve notification messages

### PR DESCRIPTION
**Pull request by holzeis**
Subject for discussion, but I think the adapted messages might be clearer and more fun. :)

|	|	|
|:-:|:-:|
| IMG_4989 | IMG_4988 | 
| IMG_4987 | IMG_4986 |

follow up to review remarks on https://github.com/get10101/10101/pull/2354

### CODE REVIEW

File: coordinator/src/notifications.rs [hunk 1]
```diff
@@ -115,20 +115,22 @@ fn build_notification(kind: &NotificationKind) -> fcm::Notification<'_> {
...
         NotificationKind::PositionExpired => {
-            notification_builder.title("Your position has expired");
-            notification_builder.body("Close your position.");
+            notification_builder.title("Your position has expired 🥴");
+            notification_builder.body("Open your app to execute the expiration.");
         }
         NotificationKind::PositionLiquidated => {
-            notification_builder.title("Your position has been liquidated");
-            notification_builder.body("Close your position.");
+            notification_builder.title("Your position has been liquidated 📉📈");
```
Comment (luckysori) [inline coordinator/src/notifications.rs:127]: The title doesn't seem to fit in the screenshot you took.

File: coordinator/src/notifications.rs [hunk 2]
```diff
@@ -115,20 +115,22 @@ fn build_notification(kind: &NotificationKind) -> fcm::Notification<'_> {
     let mut notification_builder = fcm::NotificationBuilder::new();
     match kind {
         NotificationKind::PositionSoonToExpire => {
-            notification_builder.title("Your position is about to expire");
-            notification_builder.body("Rollover your position for the next cycle.");
+            notification_builder.title("Your position is about to expire ⏳");
+            notification_builder
+                .body("Open your app to rollover your position for the next cycle.");
```
Comment (luckysori) [inline coordinator/src/notifications.rs:120]: The verb is "to roll over".

```suggestion
                .body("Open your app to roll over your position for the next cycle.");
```

File: coordinator/src/notifications.rs [hunk 3]
```diff
@@ -115,20 +115,22 @@ fn build_notification(kind: &NotificationKind) -> fcm::Notification<'_> {
...
-            notification_builder.body("Close your position.");
+            notification_builder.title("Your position has been liquidated 📉📈");
+            notification_builder.body("Open your app to execute the liquidation.");
         }
         NotificationKind::RolloverWindowOpen => {
-            notification_builder.title("Rollover window is open");
-            notification_builder.body("Rollover your position for the next cycle.");
+            notification_builder.title("Rollover window is open 🪟");
+            notification_builder
+                .body("Open your app to rollover your position for the next cycle.");
```
Comment (luckysori) [inline coordinator/src/notifications.rs:133]:
```suggestion
                .body("Open your app to roll over your position for the next cycle.");
```

**Review by luckysori — APPROVED** [review]
I like it, thanks!

File: coordinator/src/notifications.rs (hunk 1 above)
Comment (holzeis) [reply coordinator/src/notifications.rs:127]: yeah, should we remove the emojis?

Comment (luckysori) [reply coordinator/src/notifications.rs:127]: What about: "Woops, someone got liquidated 💸"

Comment (holzeis) [reply coordinator/src/notifications.rs:127]: We could probably make a specific notification depending on wether the user or the coordinator got liquidated.

Comment (holzeis) [reply coordinator/src/notifications.rs:127]: Thanks for the feedback, I will work on this tomorrow.

**Comment by holzeis** [comment]
|	|	|	|
|:-:|:-:|:-:|
| IMG_5006 | IMG_5005 | IMG_94A539E45211-1 |

**Pull request merged**